### ISSUE TITLE
Import/export config: https://github.com/ricochet-im/ricochet/issues/577

### ISSUE THREAD

**Issue by eleanor-em** [blueprint-freespeech/ricochet-refresh#1]
Should be easy enough to do: zip the config folder to export, unzip and copy to import.

**Comment by pospeselr** [comment]
Agree, we need UX here for importing/exporting the user configuration. Power users can maintain backups of their 'config' folder created on ricochet-refresh first launch,
